tests/spreadsheet/spreadsheet_component.test.ts

```diff
@@ -106,7 +106,7 @@ describe("Simple Spreadsheet Component", () => {
                 colNumber: 26,
                 rowNumber: 100,
                 cells: {
-                  A1: { content: "=GETACTIVESHEET()" },
+                  A1: "=GETACTIVESHEET()",
                 },
                 conditionalFormats: [],
               },
```

Comment:
For example here, why do we care about the number of row/columns in this test ? We don't care about the sheet name. Nor the data version. Why is there an empty array of conditional formats, but not an empty array with figures ? And did we ever had a key `activeSheet` in the model  data ? 

A PR to cleanup all that would probably be -500 lines of code at least #FPCompliance